### PR TITLE
chore(acc): Automatically execute acc test when open a pr

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
1. Automatically execute acc test when opening a pr.
    1. only execute when resource file change, exclude the test file
    2. the log will be added as a pull request comment
2. ignore  `vendor` dir in pr-size-labeler
3. move golang-ci to CI.yml

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

### CODE REVIEW

File: .github/workflows/acc-test.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: acct-test
+on: 
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  pr-acc-test:
+    if: ${{github.repository == 'huaweicloud/terraform-provider-huaweicloud' &&
```
Comment (ShiChangkuo) [inline .github/workflows/acc-test.yml]: I think we should not harding coding the repository address, suggest to remove it

File: scripts/pr_acctest_basic.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+all_files=$(git diff $1 $2 --name-only huaweicloud | grep -v "_test.go")
```
Comment (ShiChangkuo) [inline scripts/pr_acctest_basic.sh]: the diff result will be very huge if the branch based on an outdated upstream

File: .github/workflows/acc-test.yml (hunk 1 above)
Comment (hmbig2) [reply .github/workflows/acc-test.yml]: done

File: scripts/pr_acctest_basic.sh (hunk 2 above)
Comment (hmbig2) [reply scripts/pr_acctest_basic.sh]: done

**Pull request closed without merging**